backend/app/routers/records.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import Depends, APIRouter
+
+from typing_extensions import Annotated, Tuple, Dict 
+
+from .helpers.helpers import *
+
+router = APIRouter(
+    prefix = "/records",
+    tags = ["records"]
+) 
+
+@router.get("/next")
+async def next(wikipedia_articles: Annotated[dict, Depends(generate_wikipedia_articles)]):
```

Comment:
Your `/current` root route could go here.

Reply:
`/current` has been moved to the `routers.py`.